src/Rings/groebner.jl

```diff
@@ -295,7 +306,7 @@ julia> leading_coefficient(G[8])
 ```
 """
 function groebner_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I)), complete_reduction::Bool=false,
-                        algorithm::Symbol = :buchberger)
+                        algorithm::Symbol = :default)
```

Comment:
Again the docstring should be adjusted, I think?

Reply:
Yes, done.